src/resolve/resolved-dns-trust-anchor.c

```diff
@@ -503,12 +498,9 @@ static int dns_trust_anchor_dump(DnsTrustAnchor *d) {
         else {
                 _cleanup_free_ char **l = NULL, *j = NULL;
 
-                l = set_get_strv(d->negative_by_name);
-                if (!l)
+                if (set_dump_sorted(d->negative_by_name, (void***) &l, /* ret_n = */ NULL) < 0)
```

Comment:
hmm, this changes behaviour, i.e. comparison will be case sensitive, but before it was case insensitive.

Reply:
ah, ignore me, got it.